spec/features/user_visits_place_spec.rb

```diff
@@ -92,4 +76,68 @@
       end
     end
   end
+
+  describe "because im an a Capyn00b" do
+
+    before(:each) do
+      stub_omniauth
+      create_place
+      create_user
+      UserFollowsController.any_instance.stub(:current_user).
+        and_return(User.first)
+      UsersController.any_instance.stub(:current_user).
+        and_return(User.first)
+      Comment.stub(:for_a_place).
+        and_return(nil)
+    end
+
+    it "can comment" do
+      VCR.use_cassette("new place") do
+        login_user
+
+        visit '/places/turing-school-of-software-design'
+        page.fill_in 'input-comment',
+          :with => 'The food is bad'
+
+        hidden_field = find :xpath, "//input[@id='sentiment']"
+        hidden_field.set "1"
+
+        hidden_field = find :xpath, "//*[@id='comment_commentable_id']"
+        hidden_field.set "#{Place.first.id}"
+
+        hidden_field = find :xpath, "//input[@id='comment_commentable_type']"
+        hidden_field.set "Place"
+
+        click_on "Submit"
+
+        visit current_path
+
+        expect(Comment.count).to eq(1)
+      end
+    end
+
+    it "cant comment if not logged in" do
+      VCR.use_cassette("new place") do
+
+        visit '/places/turing-school-of-software-design'
+        page.fill_in 'input-comment',
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.